R/dribble.R

```diff
@@ -47,3 +53,27 @@ as.dribble.list <- function(x, ...) {
     class = c("dribble", "tbl_df", "tbl", "data.frame")
   )
 }
+
+## this let's us pull things out of drive_file column that we'd like
+## as a column in the main dribble
+pull_into_dribble <- function(dribble, pull) {
+
+  if (nrow(dribble) == 1) {
+    dribble[[pull]] <- dribble$drive_file[[1]][[pull]]
+    dribble
+  } else {
+
+    mp <- list(character = purrr::map_chr,
+               numeric = purrr::map_dbl,
+               list = purrr::map,
+               logical = purrr::map_lgl
+    )
+
+    cl <- class(dribble$drive_file[[1]][[pull]])
```

Comment:
From what I've seen so far, I think the only classes that the API returns are those (and actually just 3, character, logical, and list), but I agree we should be careful.